Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -8281,8 +8267,7 @@ class BBQJIT {
         case TypeKind::Funcref:
         case TypeKind::Structref:
         case TypeKind::Arrayref:
-            m_jit.load64(srcLocation.asAddress(), m_dataScratchGPR);
-            m_jit.store64(m_dataScratchGPR, dst.asAddress());
+            m_jit.transfer64(srcLocation.asAddress(), dst.asAddress());
```

Comment:
Should we make a `transferPtr` instead just so we can better facilitate the removal of Air later?